components/query_filter/utils.cc

```diff
@@ -117,15 +117,44 @@ static constexpr auto kConditionalQueryStringTrackers =
         {"mkt_tok", "([uU]nsubscribe|emailWebview)"},
     });
 
-static constexpr auto kScopedQueryStringTrackers =
-    base::MakeFixedFlatMapSorted<std::string_view, std::string_view>({
+// The second parameter is a comma-separated list of domains.
+// The domain comparison will also match on subdomains. So if the
+// parameter is scoped to example.com below, it will be removed from
+// https://example.com/index.php and from http://www.example.com/ for
+// example.
+static const auto kScopedQueryStringTrackers =
+    std::map<std::string_view, std::vector<std::string_view>>({
```

Comment:
probably with `std::array` we can keep the whole thing `constexpr`

Reply:
I tried to do this, but I wasn't able to make it work as a `constexpr` with an `std::array` because the size of the array is variable.